test/jit/test_tensor.py

```diff
@@ -0,0 +1,29 @@
+import torch
+from torch.testing._internal.jit_utils import JitTestCase
+
+
+class TestTensor(JitTestCase):
+    def test_numel(self):
+        @torch.jit.script
+        def get_numel_script(x):
+            return x.numel()
+
+        x = torch.rand(3, 4)
+        numel = get_numel_script(x)
+        self.assertEqual(numel, 3 * 4)
+
+    def test_element_size(self):
+        @torch.jit.script
+        def get_element_size_script(x):
```

Comment:
not really sure this warrants it's own file... we have other tests on tensor methods in test_jit.py

Reply:
Moved to test_jit.py